src/components/tabs/_tabs.scss

```diff
@@ -41,8 +39,10 @@
     @include govuk-font($size: 19);
 
     display: inline-block;
-    padding-top: govuk-spacing(2);
-    padding-bottom: govuk-spacing(2);
+    // Use bottom margin on mobile and when JS is not enabled.
+    // Consistent with what we do with on-page navigation in GOV.UK Design
```

Comment:
I think this comment can probably just be removed? I don't think we'd have written it if we'd built the component this way from scratch, and I think it's likely just to go out of date.